healthcareai/common/transformers.py

```diff
@@ -20,11 +21,12 @@ class DataFrameImputer(TransformerMixin):
     Columns of other types (assumed continuous) are imputed with mean of column.
     """
 
-    def __init__(self, impute=True, verbose=True):
+    def __init__(self, impute=True, verbose=True, max_impute=.5):
```

Comment:
- Great idea. Would you mind keeping the `verbose` arg last?
- At this point I think that this method needs a docstring. We prefer google format, and if you're using an editor like pycharm it's trivial to get that formatting right. No need to go overboard, just note what the arguments mean.